.github/workflows/rust.yml

```diff
@@ -13,6 +13,11 @@ jobs:
   build:
     runs-on: ubuntu-latest
     steps:
+      - name: Start the postgres service that is already in the runner ubuntu image
+        run: |
+          sudo systemctl start postgresql.service
+          pg_isready
+          sudo -u postgres psql -c "ALTER USER postgres WITH PASSWORD 'password'"
```

Comment:
Docker image builds with offline sqlx now, but I think I would keep the online build here, and look into verifying that the generated sqlx files match what we have in migrations.